wiki/Accuracy/en.md

```diff
@@ -1,96 +1,125 @@
-Accuracy
-==========
+<!-- wiki -->
+[osu! wikilink]: /wiki/Game_Modes/osu!/ "osu!"
+[osu!taiko wikilink]: /wiki/Game_Modes/osu!taiko/ "osu!taiko"
+[osu!catch wikilink]: /wiki/Game_Modes/osu!catch/ "osu!catch"
+[osu!mania wikilink]: /wiki/Game_Modes/osu!mania/ "osu!mania"
 
-There are three types of accuracy: per [beatmap](/wiki/Beatmaps)'s accuracy, one's overall accuracy, and [pp](/wiki/Performance_Points)'s accuracy.
+[beatmap wikilik]: /wiki/Beatmaps "Beatmaps"
+[pp wikilink]: /wiki/Performance_Points "Performance Points"
+[Score wikilink]: /wiki/Score "Score"
 
-In simple terms, the beatmap's accuracy is dependent on [Hit-scores](/wiki/Score) gained, overall accuracy is dependent on each submitted top score's accuracy (note that the accuracy will the weighted in some way so better scores influence your overall accuracy more than worse scores), and pp's accuracy is dependent on submitted online score's accuracy which is eligible for pp gain.
+# Accuracy
 
+There are three types of accuracy:-
 
-Standard
-----------
+- per [beatmap][beatmap wikilik]'s accuracy,
+- one's overall accuracy, and
+- [Performance Points (pp)][pp wikilink]'s accuracy.
 
-In [Standard](/wiki/Game_Modes/osu!), accuracy is calculated by weighting the judgement gained on each note by its value and dividing by the maximum possible amount. In other words:
+In simple terms,
 
-**Accuracy = Total points of hits / (Total number of hits * 300)**
+- the beatmap's accuracy is dependent on [Hit-scores][Score] gained,
+- overall accuracy is dependent on each submitted top score's accuracy
+  - note that the accuracy will the weighted in some way so better scores influence overall accuracy more than worse scores, and
+- pp's accuracy is dependent on submitted online score's accuracy which is eligible for pp gain.
 
-| Total points of hits | Total number of hits |
-| ---------------------| ---------------------|
-| (Number of 50s * 50 + Number of 100s * 100 + Number of 300s * 300) | (Number of misses + Number of 50's + Number of 100's + Number of 300's) |
+## Game Modes
 
-For reference: 300 = 6/6, 100 = 2/6, 50 = 1/6, Miss = 0/6.
+### osu!standard
 
-Taiko
-------
+In [osu!standard][osu! wikilink], accuracy is calculated by weighting the judgement gained on each note by its value and dividing by the maximum possible amount.
 
-In [Taiko](/wiki/Game_Modes/osu!taiko) mode, the song accuracy is calculated by sum of all note accuracy divided by number of notes. A GREAT (良) counts as 100%, a GOOD (可) as 50% (half) and a MISS/BAD (不可) as 0% (which breaks the combo). Drumrolls and spinners do not influence the accuracy.
+In other words: **Accuracy = Total points of hits / (Total number of hits * 300)**
 
-**Accuracy = Total points of hits / (Total number of hits * 300)**
+Term | Formulae
```

Comment:
`Term | Formula` or `Terms | Formulae` since using plural in only one header looks weird